test/common/network/lc_trie_speed_test.cc

```diff
@@ -97,44 +97,45 @@ BENCHMARK(BM_LcTrieLookupMinimal);
 
 } // namespace Envoy
 
-// Boilerplate main(), which discovers benchmarks in the same file and runs them.
-int main(int argc, char** argv) {
-
-  // Random test addresses from RFC 5737 netblocks
-  static const std::string test_addresses[] = {
-      "192.0.2.225",   "198.51.100.55", "198.51.100.105", "192.0.2.150",   "203.0.113.162",
-      "203.0.113.110", "203.0.113.99",  "198.51.100.23",  "198.51.100.24", "203.0.113.12"};
-  for (const auto& address : test_addresses) {
-    addresses.push_back(Envoy::Network::Utility::parseInternetAddress(address));
-  }
+// Hack to initialize the global variables used by the benchmarks above.
```

Comment:
Can't we just have a setup method per-benchmark?

Reply:
I think that would require a change to the google benchmarking libraries.  It's doable, but doesn't seem trivial.

Reply:
Why can't each benchmark method call into a setup function with the benchmark clock disabled?

Reply:
Refactored how test data is constructed.